launcher/modManager/cmodlist.cpp

```diff
@@ -176,15 +176,15 @@ void CModList::modChanged(QString modID)
 {
 }
 
-static QVariant getValue(QVariantMap input, QString path)
+static QVariant getValue(QVariant input, QString path)
```

Comment:
most Qt types are "heavy", prefer passing them by const ref

Reply:
Thus I have to refactor across whole launcher

Reply:
yeah, let's leave for another PR :)